static/js/stream_settings_ui.js

```diff
@@ -48,6 +49,16 @@ export function set_right_panel_title(sub) {
     );
 }
 
+export function update_stream_privacy_icon_in_settings(sub) {
+    $(".general_settings .large-icon").replaceWith(
```

Comment:
We need to be more selective here to update only if the section opened is of the changed stream. If you change privacy in one window and keep another window open in the browser showing settings of different stream, the icon of that incorrect stream will be updated.